### PR TITLE
reading list: Set device by flag instead of arg

### PR DESCRIPTION
**Pull request by difince**
Usually Args are reserved for commands object ID (in our case reading ID)
That is why now the device (name) is currently set by flag --device
Address partially: https://github.com/edgexfoundry-holding/edgex-cli/issues/213

Signed-off-by: Diana Atanasova <dianaa@vmware.com>

### CODE REVIEW

File: cmd/reading/list/list.go [hunk 1]
```diff
@@ -40,20 +41,20 @@ const readingTemplate = "Reading ID\tName\tDevice\tOrigin\tValue\tCreated\tModif
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
-		Short: "A list of all device readings",
-		Long:  `Return all device readings.`,
+		Short: "A list of all readings or list of all device readings",
```
Comment (mkbhanda) [inline cmd/reading/list/list.go:44]: Diana .. are we not limited to 50 or some limited number as specified by the "limit" flag of readings? "all" implies no limits. how about instead:
"A list of readings across devices or pertaining to a specified device"

Comment (difince) [reply cmd/reading/list/list.go:44]: An issue opened by you https://github.com/edgexfoundry/edgex-go/issues/2560
./edgex-cli reading list (v1/reading endpoint) does not support `limit`
meanwhile ./edgex-cli reading list <device-name> --limit 40 - does support `limit` - In this case the default value if not set is 50

Comment (difince) [reply cmd/reading/list/list.go:44]: v1/reading:
  get:
  description: Return list of all readings, sorted by reading ID.
---------------------------------------------
 /v1/reading/device/{deviceId}/{limit}:
    get:
      description: Return a list of all readings for a given device, sorted
        by the readings modified date.
------------------------------------- 
any suggestion that include in cli helper messages (short/long) these details ?

Comment (difince) [reply cmd/reading/list/list.go]: message changed to "A list of readings across devices or pertaining to a specified device"

**Pull request merged**